filament/src/driver/opengl/OpenGLDriver.cpp

```diff
@@ -2368,23 +2368,23 @@ GLuint OpenGLDriver::getSamplerSlow(driver::SamplerParams params) const noexcept
 }
 
 void OpenGLDriver::insertEventMarker(char const* string, size_t len) {
-#if GL_EXT_debug_marker
+#ifdef GL_EXT_debug_marker
```

Comment:
Is ifdef enough? I don't remember if GL always #defines tokens or if it sets them to 0/1

Reply:
They are inconsistent about this, so for robust builds we should #ifdef, especially since we do a run-time check for the actual extension anyway.

Reply:
Right but I want to make sure we don't end up taking the path because the token is defined and set to 0.

Reply:
glext sets these 1, it never sets these to 0.